docs/faq.rst

```diff
@@ -406,3 +406,16 @@ Example: ::
         await ctx.send(f'Pushing to {remote} {branch}')
 
 This could then be used as ``?git push origin master``.
+
+Application Commands
+--------------------
+
+Questions regarding Discord's new Application Commands.
+
+My bot's commands are not showing up!
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Did you :meth:`~.CommandTree.sync` your command? Commands need to be synced before they will appear.
+2. Did you invite your bot with the correct permissions? Bots need to be invited with the ``application.commands``
+   scope in addition to the ``bot`` scope. For example, invite the bot with the following URL:
+   https://discord.com/oauth2/authorize?client_id=<client id>&scope=applications.commands+bot
```

Comment:
```suggestion
   ``https://discord.com/oauth2/authorize?client_id=<client id>&scope=applications.commands+bot``
```

This should also probably mention that `utils.oauth_url` helps create this URL too.